gremlin-server/pom.xml

```diff
@@ -148,6 +149,30 @@ limitations under the License.
     </build>
     <profiles>
         <profile>
+            <id>linux</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>EPOLL</name>
```

Comment:
Should add a note in the contributing doc about the `EPOLL` profile, similar to `includeNeo4j` https://github.com/apache/incubator-tinkerpop/blob/master/CONTRIBUTING.asciidoc